casper/src/test/scala/coop/rchain/casper/helper/HashSetCasperTestNode.scala

```diff
@@ -55,7 +55,8 @@ class HashSetCasperTestNode[F[_]](
     val blockDagDir: Path,
     val blockStoreDir: Path,
     blockProcessingLock: Semaphore[F],
-    shardId: String = "rchain"
+    shardId: String = "rchain",
+    createRuntime: (Path, Long) => (RuntimeManager[Task], () => Task[Unit])
```

Comment:
Shouldn't this be `(Path, Long) => Resource[Task, RuntimeManager[Task]]`?

Reply:
compromise deployed